Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:2.7.11
+FROM python:2.7
```

Comment:
@hyi - currently 2.7.13 is the most recent version, by using 2.7 it will get the most recent version in the 2.7.x tree. 
This is somewhat at the mercy of the entity creating the images, but Python has chosen to follow the practice of highest sub-level available if it's not explicitly defined by the user.

Tags 2, 2.7 and 2.7.13 would all yield the same image being pulled ([reference](https://hub.docker.com/_/python/))